src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -0,0 +1,255 @@
+   */
+  const remainingToStakeSafe =
+    remainingToStake > 0
+      ? Math.round(remainingToStake * 100) / 100
+      : remainingToStake;
+  /*
+   * This basically doubles as the user's reputation
+   * So we can use it to also check if the user can actually stake
+   * If the reputation is 0, they cannot stake at all
+   */
+  const userStakeTopLimit = parseFloat(
+    moveDecimal(
+      maxUserStake,
+      -1 * getTokenDecimalsWithFallback(nativeToken?.decimals),
+    ),
+  );
+  const userStakeBottomLimit = parseFloat(
+    moveDecimal(
+      minUserStake,
+      -1 * getTokenDecimalsWithFallback(nativeToken?.decimals),
+    ),
+  );
+
+  const canUserStake =
+    hasRegisteredProfile &&
+    bigNumberify(maxUserStake).gt(0) &&
+    bigNumberify(maxUserStake).gte(bigNumberify(minUserStake));
+
+  return (
+    <div className={styles.main}>
+      <ActionForm
+        initialValues={{
+          amount: userStakeBottomLimit,
+        }}
+        validationSchema={validationSchema}
+        submit={ActionTypes.COLONY_MOTION_STAKE}
+        error={ActionTypes.COLONY_MOTION_STAKE_ERROR}
+        success={ActionTypes.COLONY_MOTION_STAKE_SUCCESS}
+        transform={transform}
+        onSuccess={handleSuccess}
+      >
+        {({ values }) => (
+          <div className={styles.wrapper}>
+            <div className={styles.title}>
+              <Heading
+                text={MSG.title}
+                className={styles.title}
+                appearance={{ size: 'normal', theme: 'dark', margin: 'none' }}
+              />
+              <QuestionMarkTooltip
+                tooltipText={MSG.stakingTooltip}
+                className={styles.help}
+                tooltipClassName={styles.tooltip}
+                tooltipPopperProps={{
+                  placement: 'right',
+                }}
+              />
+            </div>
+            <p className={styles.description}>
+              <FormattedMessage {...MSG.description} />
+            </p>
+            <span className={styles.amount}>{`${parseFloat(
+              values.amount,
+            ).toFixed(2)} ${nativeToken?.symbol}`}</span>
+            <div className={styles.sliderContainer}>
+              <Slider
+                name="amount"
+                value={values.amount}
+                min={userStakeBottomLimit}
+                max={
+                  remainingToStakeSafe < userStakeBottomLimit
+                    ? userStakeBottomLimit
+                    : remainingToStakeSafe
+                }
+                limit={userStakeTopLimit}
+                step={0.01}
+                disabled={!canUserStake || remainingToStakeSafe === 0}
+              />
+            </div>
+            <div className={styles.buttonGroup}>
+              <Button
+                appearance={{ theme: 'primary', size: 'medium' }}
+                type="submit"
+                disabled={!canUserStake || remainingToStakeSafe === 0}
```

Comment:
You cannot stake less than than the `minUserStake` since this is enforced by the contracts, so both the values and display are correct.

But what happens underneath the hood, is that the contract only takes enough to fully stake the motion and returns the rest to you (yes, you can over-stake), so in a sense you are correct, that you are "only" staking `remainingToStakeSafe`, while sending the transaction for the `minUserStake` amount

This is the correct implementation.